openapi/api.yml

```diff
@@ -85,6 +85,93 @@ paths:
                 format: binary
                 description: The file content in a binary format (e.g. XLSX).
 
+  /async_download:
+    get:
+      operationId: core.controllers.async_download.async_download
+      summary: Get monitoring and evaluation data from the data-store, store report and email link.
+      description: Query the database with the provided parameters and serialise the resulting data in the specified format.
+      tags:
+        - main
+      parameters:
+        - in: query
+          name: email_address
+          description: email address to send the file to
+          example: "john.smith@levellingup.gov.uk"
+          schema:
+            type: string
+          required: true
+        - in: query
+          name: file_format
+          description: file format of serialised data
+          example: "xlsx"
+          schema:
+            type: string
+            enum: [ json, xlsx ]
+          required: true
+        - in: query
+          name: funds
+          example: [ "HS" ]
+          description: filter by fund ids
+          schema:
+            type: array
+            items:
+              type: string
+        - in: query
+          name: organisations
+          example: [ "2a8baca6-5203-4fa5-be23-8b7797f111f7" ]
+          description: filter by organisation (UUID)
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+        - in: query
+          name: regions
+          example: [ "TLC", "TLD", "TLK" ]
+          description: filter by region (ITL code)
+          schema:
+            type: array
+            items:
+              type: string
+        - in: query
+          name: rp_start
+          description: filter by reporting period start (ISO8601 format)
+          example: "2023-04-01T00:00:00Z"
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: rp_end
+          description: filter by reporting period end (ISO8601 format)
+          example: "2024-03-31T00:00:00Z"
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: outcome_categories
+          description: filter by outcome category
+          example: [ "Transport" ]
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  file_content:
+                    type: object
+                    description: The file content in json format.
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+                description: The file content in a binary format (e.g. XLSX).
```

Comment:
I don't think this is right, is it? The `async_download` view doesn't return any data as far as I can see?

Reply:
Not sure if it's the correct response, but changed it to:
responses:
        '204':
          description: Request has been completed successfully.